File: JAVA PROGRAMMING/Test24.java
// 24. PRINT SUM OF ALL NUMBERS BETWEEN RANGE1 TO RANGE2
import java.util.Scanner;
public class Test24 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Range1 Value: ");
        int range1 = in.nextInt();

        System.out.println("Enter Range2 Value: ");
        int range2 = in.nextInt();

        int sum = 0;
        for(int i = range1; i <= range2; i++) {
            sum += i;
        }
        System.out.println("Sum Of "+range1+" To "+range2+" Is: "+sum);
        in.close();
    }
}

File: JAVASCRIPT PROGRAMMING/Test24.js
"use strict";
// 24. PRINT SUM OF ALL NUMBERS BETWEEN RANGE1 TO RANGE2
const func24 = (range1, range2) => {
    let sum = 0;
    for (let i = range1; i <= range2; i++) {
        sum += i;
    }
    console.log(`Sum: ${sum}`);
};
func24(10, 20);
func24(10, 50);
func24(0, 100);